fractions/microprofile/microprofile-jwt/src/main/java/org/wildfly/swarm/microprofile/jwtauth/MicroProfileJWTAuthFraction.java

```diff
@@ -60,6 +62,31 @@
     @Configurable("thorntail.microprofile.jwt.default-missing-method-permissions-deny-access")
     private Defaultable<Boolean> defaultMissingMethodPermissionsDenyAccess = bool(true);
 
+    /**
+     * Realm name
+     */
+    @Configurable("thorntail.microprofile.jwt.realm")
+    @Configurable("thorntail.microprofile.jwtauth.realm")
```

Comment:
Can remove this one. We only have `jwtauth` for backwards compatibility, `jwt` is the correct thing to do.

Reply:
Ah please ignore, I see it was copied from somewhere else.